### ISSUE TITLE
episode titles are vague

### ISSUE THREAD

**Issue by colinsauze** [carpentries-incubator/deep-learning-intro#218]
Split off from issue #199:

> Another argument would be that the episode titles "Monitor the training process" and "Networks are like onions" are vague, and more concrete titles would both help learners navigate the lesson before, during and after a workshop, and look better in a schedule on a workshop webpage so that potential participants get an idea about what the workshop contains.

Personally I think that "Monitor the training process" is a reasonably good description of episode 3. But I agree "Networks are like onions" is vague. How about changing that to "Types of Layers and Dropout", "Optimising image classifying networks" or just "Networks for Image Classification"?

**Comment by dafnevk** [comment]
Note that the questions of the episode are:
* Why do we need different types of layers?
* What are good network designs for image data?
* What is a convolutional layer?
* How can we avoid overfitting?

So I would argue that both the aspects of layer types and image data could be reflected in the title.
So maybe 'Network layers for image classification' ?

**Comment by dafnevk** [comment]
Maybe having 'image classification' in the title discourages learners who are not interested in image data. We can focus more on the fact that different layer types are introduced (convolutional, dropout, etc), more advanced than fully connected layers: ''Advanced layer types".
